ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction.proto

```diff
@@ -101,6 +101,7 @@ message Transaction {
 
   // The collection of events.
   // Only contains ``CreatedEvent`` or ``ArchivedEvent``.
+  // Omits events on transient contracts, i.e., contracts that were both created and archived in this transaction.
```

Comment:
Just to be safe, this is probably better to be left on the service, rather than the message.

Reply:
Sorry for the delay, slipped through the cracks: does this look better?

Reply:
It does, thanks. I approved with one remark.